modules/serverless-pingsource-ODC.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-pingsource.adoc
+
+[id="serverless-pingsource-ODC_{context}"]
+= Creating a PingSource using {product-title} web console
+
+The following sections describe how to create and verify a basic PingSource from the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+
+.Procedure
+
+. To verify that the PingSource is working, create a simple Knative
+service that dumps incoming messages to the service's logs.
+.. In the *Developer* perspective, navigate to *+Add* -> *YAML*.
+.. Copy the example YAML:
++
+
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+
+.. Click *Create*.
+
+. Create a PingSource in the same namespace as the service created in the previous step, or any other sink that you want to send events to.
+.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*.
+.. Select *Ping Source*.
+.. Enter a value for *Schedule*. In this example, the value is `*/2 * * * *`, which will create a PingSource that sends a message every two minutes.
+.. Optional. You can enter a value for *Data*, which will be the message payload.
```

Comment:
I see Optional rendered like this `Optional:` here: https://docs.openshift.com/container-platform/4.4/networking/hardware_networks/configuring-sriov-device.html though I don't see this in the OCP guidelines.